src/tracker/tracker-funcs.jl

```diff
@@ -497,3 +497,58 @@ function addfloemasks!(props, imgs)
     end
     return nothing
 end
+
+## LatLon functions originally from IFTPipeline.jl
```

Comment:
Will it be removed from `ice-floe-tracker-pipeline`? It looks like these were orginially in the `h5.jl` script in that repo.

Reply:
https://github.com/WilhelmusLab/ice-floe-tracker-pipeline/pull/55

Reply:
https://github.com/WilhelmusLab/ice-floe-tracker-pipeline/issues/90